### PR TITLE
Updating documentation

### PR DESCRIPTION
**Pull request by benjefferies**
### What is the context of this PR?
* Updating missing documentation in readme
* Adding aws profile support
* Adding more non-sensitive variables to template

### How to review
Describe the steps required to test the changes.

- [ ] Do all commits have sensible commit messages?
- [ ] Is there documentation or is the code self-describing?
- [ ] Test by deploying environment

### CODE REVIEW

File: survey-runner/terraform.tfvars.example [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
-application_cidr="10.30.2X.0/26"
-database_1_cidr="10.30.2X.64/26"
-database_2_cidr="10.30.2X.128/26"
+application_cidr="10.30.20.0/26"
+database_1_cidr="10.30.20.64/26"
+database_2_cidr="10.30.20.128/26"
 dev_mode="True"
 rsyslogd_server_ip="10.172.x.x"
-schema_bucket="eq-schema-files-preprod"
+schema_bucket="eq-schema-files-XXXXX" # Set to your schema bucket
```
Comment (collisdigital) [inline survey-runner/terraform.tfvars.example]: Is this use anymore, now that S3 has been removed?

Comment (benjefferies) [reply survey-runner/terraform.tfvars.example]: Not sure if it's being used for the author but can probably go if author in its current form is dead

File: survey-runner/run-terraform.sh [hunk 2]
```diff
@@ -1,25 +1,30 @@
 #!/bin/bash
 
+if [ -n "$AWS_DEFAULT_PROFILE" ]; then
```
Comment (ajmaddaford) [inline survey-runner/run-terraform.sh]: I think `AWS_DEFAULT_PROFILE` confuses this with the the default AWS profile you when you don't provide a profile to the aws cli, any reason not to use `AWS_PROFILE`?

File: survey-runner/terraform.tfvars.example [hunk 3]
```diff
@@ -1,27 +1,26 @@
...
+elastic_beanstalk_aws_key_pair="XXXXX" # Set to the name generated ssh key in aws
+ons_access_ips=[XXXXXXX, XXXXXXX] # Set to the IP addresses you will be accessing accessing survey runner from
+certificate_arn="arn:aws:iam::XXXXXXX:server-certificate/NAME" # Set to arn of the tls certificate used by survey runner
 application_secret_key="you'll never guess it"
 vpc_ip_block="10.30.20.0/24"
-rabbitmq_ip_prime="XX.XX.XX.15"
-rabbitmq_ip_failover="XX.XX.XX.16"
+rabbitmq_ip_prime="10.30.20.15"
+rabbitmq_ip_failover="10.30.20.16"
 google_analytics_code="UA-56892037-7"
```
Comment (ajmaddaford) [inline survey-runner/terraform.tfvars.example]: I think the default value for `google_analytics_code` should be `""`. That should mean that eq-survey-runner doesn't setup analytics (https://github.com/ONSdigital/eq-survey-runner/blob/master/app/__init__.py#L113)
Also it's duplicated below set to `XXXXXXX`

**Review by ajmaddaford — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -26,29 +26,43 @@ Also the pre-prod.pem key must exist in this directory.
...
+
+1. Move the ssh key to `eq-terraform/survey-runner`
+
+1. Copy `terraform.tfvars.example` to `terraform.tfvars` in both author and survey-runner. You'll need to change all values containing an 'X' to match your requirements, including the AWS credentials you set up previously.
+
+1. To deploy survey-runner:
+
+  - Export an `AWS_ENVIRONMENT_NAME` environment variable e.g. `export AWS_ENVIRONMENT_NAME=preprod` or `export AWS_ENVIRONMENT_NAME=$USER`
+
+  - If using awscli profile export an `AWS_DEFAULT_PROFILE` environment variable e.g. `export AWS_DEFAULT_PROFILE=ons`
```
Comment (jonnyshaw89) [inline README.md]: probably need to add a note that this step is only required if running multiple AWS profiles

File: survey-runner/terraform.tfvars.example (hunk 3 above)
Comment (jonnyshaw89) [reply survey-runner/terraform.tfvars.example]: Agreed, this will taint any analytics received otherwise

**Review by jonnyshaw89 — CHANGES_REQUESTED** [review]

File: survey-runner/terraform.tfvars.example (hunk 3 above)
Comment (benjefferies) [reply survey-runner/terraform.tfvars.example]: Done

File: survey-runner/run-terraform.sh (hunk 2 above)
Comment (benjefferies) [reply survey-runner/run-terraform.sh]: Removed this as it's not needed if you set the default profile

File: README.md (hunk 4 above)
Comment (benjefferies) [reply README.md]: Done

**Review by jonnyshaw89 — APPROVED** [review]

**Review by ajmaddaford — APPROVED** [review]

**Pull request merged**